`load_description()` can't load descriptions for previous working ones

There are two runs with an hour or two in between, today: [here](https://github.com/kreshuklab/plant-seg/actions/runs/11962866393/job/33352160971) and [here](https://github.com/kreshuklab/plant-seg/actions/runs/11961183045/job/33346839000?pr=361).

```pytb
self = <plantseg.core.zoo.ModelZoo object at 0x7fa99c6d2540>
model_id = 'efficient-chipmunk'
    def get_model_by_id(self, model_id: str):
        """Load model from BioImage.IO Model Zoo.
    
        E.g. model_id = 'efficient-chipmunk', whose RDF URL is:
        https://bioimage-io.github.io/collection-bioimage-io/rdfs/10.[52](https://github.com/kreshuklab/plant-seg/actions/runs/11962866393/job/33352160971#step:11:53)81/zenodo.8401064/8429203/rdf.yaml
        """
    
        if not self._bioimageio_zoo_all_model_url_dict:
            self.refresh_bioimageio_zoo_urls()
    
        if model_id not in self._bioimageio_zoo_all_model_url_dict:
            raise ValueError(f"Model ID {model_id} not found in BioImage.IO Model Zoo")
    
        rdf_url = self._bioimageio_zoo_all_model_url_dict[model_id]
        model_description = load_description(rdf_url)  # FIXME: HERE - I think it's an issue from Bioimage.IO Spec
    
        # Check if description is `ResourceDescr`
        if isinstance(model_description, InvalidDescr):
            model_description.validation_summary.display()
>           raise ValueError(f"Failed to load {model_id}")
E           ValueError: Failed to load efficient-chipmunk
plantseg/core/zoo.py:343: ValueError
----------------------------- Captured stdout call -----------------------------
                                                                                
  ❌                bioimageio format validation failed                         
 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 
  source            https://bioimage-io.github.io/collection-bioimage-io/rdfs…  
  format version    model 0.4.9                                                 
  bioimageio.spec   0.5.3.5                                                     
                                                                                
                                                                                
  ❓   location                    detail                                       
 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 
  ✔️    .                           initialized InvalidDescr to describe model   
                                   unknown                                      
  ❌   .                           bioimageio.spec format validation model      
                                   0.4.10                                       
  🔍   context.perform_io_checks   True                                         
  🔍   context.root                https://bioimage-io.github.io/collection-b…  
  🔍   context.warning_level       error                                        
  ❌   .                           ModelDescr is not fully defined; you should  
                                   define Path, then call                       
                                   ModelDescr.model_rebuild().For further       
                                   information visit                            
                                   https://errors.pydantic.dev/2.10/u/class-n…  
                                   common_nodes.py, line 414, in _load_impl     
                                   rd = cls.model_validate(data)                
                                   ^^^^^^^^^^^^^^^^^^^^^^^^node.py, line 66,    
                                   in model_validate    return                  
                                   super().model_validate(                      
                                   ^^^^^^^^^^^^^^^^^^^^^^^main.py, line [62](https://github.com/kreshuklab/plant-seg/actions/runs/11962866393/job/33352160971#step:11:63)2,    
                                   in model_validate    return                  
                                   cls.__pydantic_validator__.validate_python(  
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^…  
                                   line 100, in getattr    raise                
                                   PydanticUserError(self._error_message,       
                                   code=self._code)
```

--- a/bioimageio/spec/_description.py
+++ b/bioimageio/spec/_description.py
@@ -164,7 +164,21 @@ def validate_format(
     format_version: Union[Literal["discover", "latest"], str] = DISCOVER,
     context: Optional[ValidationContext] = None,
 ) -> ValidationSummary:
-    """validate a bioimageio.yaml file (RDF)"""
+    """Validate a dictionary holding a boiimageio description
+    (see `bioimagieo.spec.load_description_and_validate_format_only`
+    to validate a file source).
+
+    Args:
+        data: dict holding raw bioimageio.yaml content
+        format_version: format version to (update to and) use for validation
+        context: validation context, see `bioimagieo.spec.ValidationContext`
+
+    Note: Use `bioimagieo.spec.load_description_and_validate_format_only` to validate a
+        file source instead of loading the YAML content and createing the appropriate
+        `ValidationContext`.
+        Or use `bioimagieo.spec.load_description` and access the `validation_summary`
+        attribute of the returned object.
+    """
     with context or validation_context_var.get():
         rd = build_description(data, format_version=format_version)
 

--- a/bioimageio/spec/summary.py
+++ b/bioimageio/spec/summary.py
@@ -358,7 +358,7 @@ class ValidationSummary(BaseModel, extra="allow"):
                     [
                         "🐍",
                         format_loc(d.loc),
-                        "conda compare ({d.name}):<br>"
+                        f"conda compare ({d.name}):<br>"
                         + d.conda_compare.replace("\n", "<br>"),
                     ]
                 )

--- a/setup.py
+++ b/setup.py
@@ -37,7 +37,7 @@ _ = setup(
         "packaging>=17.0",
         "pooch>=1.5,<2",
         "pydantic-settings>=2.5,<3",
-        "pydantic>=2.7.0,<3",
+        "pydantic>=2.7.0,<2.10",
         "python-dateutil",
         "requests",
         "rich",

